src/frontend/src/optimizer/plan_node/logical_scan.rs

```diff
@@ -86,7 +86,7 @@ impl LogicalScan {
     pub fn create(
         table_name: String, // explain-only
         table_desc: Rc<TableDesc>,
-        indexes: Vec<Rc<TableDesc>>,
+        indexes: Vec<(String, Rc<TableDesc>)>,
```

Comment:
Why table name (or index name) is not in `TableDesc`? cc. @st1page 

PS. `ColumnDesc` contains `name` of the column

Reply:
it seems that the compute node does not need to know the table name. and the `ColumnDesc` name is just for debug now